### DISCUSSION TITLE
Master-Detail with includ together

### DISCUSSION THREAD

**Discussion by Kakovisk** [Budibase/budibase#9667] [Help] [answered]
Hi Everyone!!

I need make a screen Master-detail for include. The includ is all together. I dont finded references.

I thinked to use temporary table but I also not do get id in Form when Type is Create.

Thanks.

**Comment by melohagan** [comment]
Hey @Kakovisk 

I don't understand the question to be honest!

**Comment by Kakovisk** [comment, marked as answer]
I have two tables. Example:

Employee and Accounts.

![Model](https://user-images.githubusercontent.com/112348859/219215708-032757da-eb9e-46a4-b9e7-be94d9c36aca.png)

The user fills in form Employee and also include many Accounts for this employee. After fill all informations (Employee and your Accounts) I save in Database.

The problem is if I use the Form in Type Create in Accounts  I havent the id of Employee.

I hope I've managed to make myself understood.

Tks

**Reply by melohagan** [reply]
Hey @Kakovisk 

I think I understand you now! There's a couple of examples in the docs on how to save two new rows with a link between each other.

 - https://docs.budibase.com/docs/form-steps
 - https://docs.budibase.com/docs/chaining-actions#example-creating-and-linking-two-table-entries-in-one-form

**Reply by Kakovisk** [reply]
This important item of documentation went unnoticed by me. I'm so sorry. Thank you for helping me.

**Reply by melohagan** [reply]
You're welcome! 😄
